test/integration/targets/user/tasks/test_create_user.yml

```diff
@@ -36,10 +36,10 @@
 
 - name: run existing user check tests
   user:
-    name: "{{ user_names.stdout_lines | random }}"
+    name: '{{ user_names.stdout_lines | random }}'
     state: present
     create_home: no
-  loop: "{{ range(1, 5+1) | list }}"
+  loop: '{{ range(1, 5+1) | list }}'
```

Comment:
The quotes here and the line above also should not be changed in this PR.

Reply:
Sorry my bad. My Prettier was running in the background at that time.